testsuite/integration/elytron-oidc-client/src/test/resources/org/wildfly/test/integration/elytron/oidc/client/deployment/OidcWithMultipleScopes.json

```diff
@@ -0,0 +1,11 @@
+{
+    "realm" : "WildFly",
+    "resource" : "MultipleScopeApp",
```

Comment:
We should use client-id and provider-url instead of realm, resource, and auth-server-url.